app/lib/dartdoc/backend.dart

```diff
@@ -47,6 +47,7 @@ class DartdocBackend {
   final DatastoreDB _db;
   final Bucket _storage;
   final VersionedJsonStorage _sdkStorage;
+  final _gcTasks = <_GCTask>{};
```

Comment:
Document that we are aware that if the server crashed pending GC tasks will disappear, and argue why this acceptable.

Presumably, new dartdoc will be generated in the future, which will trigger another GC for the package/version, then hopefully we'll catch up on the files.